store/src/main/scala/geotrellis/store/util/IOUtils.scala

```diff
@@ -81,8 +81,13 @@ object IOUtils {
         .parJoin(threads)
         .compile
         .toVector
-        .unsafeRunSync
+        .attempt
+        .unsafeRunSync match {
+          case Right(succ) => succ
+          case Left(err) => throw err
+        }
         .flatten
```

Comment:
oops, this won't work; the type had changed

Reply:
The type is the same and it is the compiler that is confused! (I believe it just needs some parens)

Reply:
@moradology :+1: makes sense